corehq/apps/hqwebapp/templates/hqwebapp/downgrade_modal.html

```diff
@@ -0,0 +1,38 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div class="modal" id="downgrade-modal" data-slug="downgrade_pending" data-domain="{{ domain }}">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header {% if days_until_downgrade <= 30 %}bg-danger text-danger{% endif %}">
+                <button type="button" class="close" data-dismiss="modal">
+                    <span aria-hidden="true">×</span>
+                    <span class="sr-only">Close</span>
+                </button>
+                <h4 class="modal-title">
+                    Your invoice for {{ invoice_month }} is past due!
+                </h4>
+            </div>
+            <div class="modal-body">
+                <p>
+                    You're at risk of losing access to paid features and priority support if your invoice isn't paid in
+                    {{ days_until_downgrade }} days.
+                </p>
+                <div class="row">
```

Comment:
Now that these buttons aren't big, it doesn't make sense to space them with the `col-sm-6` cells. I'd just use the standard `modal-footer` markup, example here: https://www.commcarehq.org/styleguide/molecules/#molecules-modals

Reply:
Hmm ran into some issues.  I'm going to revisit this.

Reply:
Grab me if you want another set of eyes on it. I hesitate to say CSS is a blocking issue but it is rather inconsistent.

Reply:
👍

Reply:
I had this on my backlog ... but I've accepted I'm not going to get to this.